hooks/pre/12-clear_pulp_content.rb

```diff
@@ -0,0 +1,73 @@
+require 'fileutils'
+
+def clear_pulp_content
+  content_dir = '/var/lib/pulp/content'
+  if File.directory?(content_dir)
+    logger.debug "Removing Pulp content inside '#{content_dir}'."
+    FileUtils.rm_rf(Dir.glob(File.join(content_dir, '*')))
+    logger.debug "Removed Pulp content inside '\#{content_dir}\', now resetting Pulp Repo importers."
+    reset_repo_importers
+    logger.info 'Pulp content successfully removed.'
+  else
+    logger.warn "Pulp content directory not present at \'#{content_dir}\'."
+  end
+end
+
+def remote_host?(hostname)
+  !['localhost', '127.0.0.1', `hostname`.strip].include?(hostname)
+end
```

Comment:
This may make sense in the shared context since it's copied from the reset data hook.